packages/core/strapi/lib/services/entity-service/types/result.d.ts

```diff
@@ -2,24 +2,63 @@ import type { Attribute, Common, EntityService, Utils } from '@strapi/strapi';
 
 type Pagination = { page: number; pageSize: number; pageCount: number; total: number };
 
-export type Entity<TSchemaUID extends Common.UID.Schema> = Utils.Object.DeepPartial<
-  GetValues<TSchemaUID>
+export type Entity<
```

Comment:
There are several places we use Entity<> without a second parameter that need to be updated, or else this shoudl have a default that includes all attributes.